drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -126,13 +126,24 @@ def self.options_for_type(pick_list_type, user:, relation_id: nil)
         Hmis::File.all_available_tags.map do |tag|
           {
             code: tag.id,
-            label: "#{tag.name} (included: #{tag.included_info})",
+            label: tag.name,
             group_code: tag.group,
             group_label: tag.group,
+            secondary_label: "(includes: #{tag.included_info})",
           }
         end.
           compact.
           sort_by { |obj| [obj[:group_label] + obj[:label]].join(' ') }
+      when 'CLIENT_ENROLLMENTS'
+        client = Hmis::Hud::Client.viewable_by(user).find_by(id: relation_id)
+        return [] unless client.present?
+
+        client.enrollments.map do |enrollment|
+          {
+            code: enrollment.id,
+            label: "#{enrollment.project.project_name} (#{[enrollment.entry_date.strfdate('mm/dd/yyyy'), enrollment.exit_date&.strfdate('mm/dd/yyyy')].compact.join(' - ')})",
```

Comment:
This might be better as:
```suggestion
            label: "#{enrollment.project.project_name} (#{[enrollment.entry_date.strfdate('mm/dd/yyyy'), enrollment.exit_date&.strfdate('mm/dd/yyyy') || 'ongoing'].join(' - ')})",
```